### ISSUE TITLE
Old Android Version

### ISSUE THREAD

**Issue by 0xdad0** [anlalalu/Q-testing#6]
I am trying your tool, but I have a concern.

The system requirements are very restrictive. The required android version (4.4) is too old. Currently, only 1.4% of android devices have android 4.4, and very few apps run on these devices. 

Is it possible to update the tool that can support newer Android versions and, therefore, more recent applications? Or at least have the source code?

Thank you.

**Comment by xixiAndpipi** [comment]
Certainly, we are updating Q-testing to support new Android versions, as soon as possible. Now, although not throughly tested, Q-testing can be used on devices of Android version 9, maybe you can have a try.
